scripts/startup.py

```diff
@@ -0,0 +1,33 @@
+import os, subprocess, locale
+
+cwd = os.getcwd()
+encoding = locale.getdefaultlocale()[1]
+
+def cmd(execute):
+    return subprocess.call(execute.format(cwd).split())
+
+if not os.path.isfile('./config/local_settings.py'):
+    cmd('cp {}/config/local_settings.py.example ./config/local_settings.py')
+if not os.path.isfile('./config/secrets.py'):
+    cmd('cp {}/config/secrets.py.example ./config/secrets.py')
+if not os.path.isfile('./config/dev_secrets.py'):
+    cmd('cp {}/config/dev_secrets.py.example ./config/dev_secrets.py')
+
+
+if not os.path.isdir('./env'):
+    cmd('virtualenv -p python3 env')
+
+def shell_source(script):
```

Comment:
Same comment on whitespace splitting for subprocess.call.

Also instead of sourcing `env/bin/activate` (and the `shell=True` and environment updates) we can simply prepend `env/bin/` to the PATH environment variable. That's supposed to be the only goal of the `activate` script, and I've used the PATH approach successfully on Linux.